packages/manager/fragments/sidebar/src/sidebar.service.ts

```diff
@@ -0,0 +1,70 @@
+import { groupBy } from 'lodash-es';
+import { getConfig } from '@ovh-ux/manager-sidebar-config';
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+import { SidebarElement } from './model/SidebarElement';
+import { Sidebar } from './model/Sidebar';
+import { ConfigItem } from './model/ConfigItem';
+import { Service } from './model/Service';
+
+
+const getPaths = (config: ConfigItem[]): string[] => config.flatMap((item) => [
+    ...(item.apiPath ? [item.apiPath] : []),
+    ...(item.subitems ? getPaths(item.subitems) : []),
+  ]);
+
+export const getServices = async (
+  paths: string[],
+): Promise<Record<string, Array<Service>>> => {
+  let services = null;
+  try {
+    services = await (
+      await fetch(`/engine/apiv6/services?routes=${paths.join(',')}`, {
+        headers: new Headers({
+          'Content-Type': 'application/json;charset=utf-8',
+          Accept: 'application/json',
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Size': '50000',
```

Comment:
Are you sure about this value?

```suggestion
          'X-Pagination-Size': '50000',
```

Seems to be used like this:

```sh
$ git grep "X-Pagination-Size"
packages/manager/apps/dedicated/client/app/billing/main/history/legacy/billing-main-history.controller.js:            'X-Pagination-Size': '5000',
packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.service.js:          'X-Pagination-Size': 5000,
```

Reply:
I may have accidentally added an extra 0 😅